extension/src/components/toggler.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { connect } from "react-redux";
+import { toggle } from "actions.js";
+import { withI18n } from "i18n.js";
+import { countUnratedRatings } from "utils.js";
+import { ToggleType } from "constants.js";
+import { Ads, UnratedRatings } from "components/ads.jsx";
+
+// Controls which section of tabs to show, defaults to the user's ads
+export const Toggle = withI18n(
+  ({ getMessage, type, message, active, amount, dispatch }) => (
+    <div
+      className={"toggle" + (active === type ? " active" : "")}
+      onClick={() => dispatch(toggle(type))}
+    >
+      {getMessage(message)}
+      {amount ? <b>{100 > amount ? amount : "100+"}</b> : ""}
```

Comment:
Is it right that this'll never happen, since the maximum number of ads saved got lowered to 50 (for both Ratings and Ads) as an attempt at a performance fix? Should we maybe move the limit back to 100?

Reply:
Yeah I think so

Reply:
We should probably make it 150 or something